kiosk/src/Components/ScanQR.tsx

```diff
@@ -50,26 +70,29 @@ const ScanQR: React.FC<IProps> = ({ kiosk }) => {
 
     return (
         <div className="scanQrPage">
-            <h2>Add game to </h2>
-            <h1>Kiosk {kioskId}</h1>
+            <h2>Add game to<br/>Kiosk {kioskId}</h2>
             <div className="scanInstructions">
-                <div className="qrOption">
-                    <p className="scanIntro">Scan game's share QR code with the button below</p>
-                    {
-                        !scannerVisible &&
-                        <button className="scanQrButton" onClick={renderQrScanner} >Scan QR code</button>
-                    }
-                    <div id="qrReader"></div>
-                    {
-                        scannerVisible &&
-                        <p className="scanTip">Tip: Do not use the kiosk's QR code</p>
-                    }
-                </div>
-
                 {
-                    !phoneWidth &&
+                    !linkVisible &&
+                    <div className="qrOption">
+                        <p className="scanIntro">Scan game's share QR code with the button below</p>
+                        {
+                            !scannerVisible &&
+                            <button className="scanQrButton" onClick={renderQrScanner} >Scan QR code</button>
+                        }
+                        <div id="qrReader" ref={qrReaderRendered}></div>
+                        {
+                            scannerVisible &&
+                            <div className="scanning">
+                                <button className="scanQrButton" onClick={stopQrScanner} >Cancel Scan</button>
+                                <p className="scanTip">Tip: Do not use the kiosk's QR code</p>
+                            </div>
+                        }
+                        <p>OR</p>
+                    </div>
+                }
+                {
                     <div className="linkOption">
-                        <p>Can't scan?</p>
                         <label>Submit share link here</label>
```

Comment:
```suggestion
                        <label>Submit share link</label>
```

I think 'here' is unnecessary and this way the label fits on one line.